src/Compiler/Microsoft.AspNetCore.Razor.Language/src/RazorLanguageVersion.cs

```diff
@@ -28,7 +28,9 @@ public sealed class RazorLanguageVersion : IEquatable<RazorLanguageVersion>, ICo
 
     public static readonly RazorLanguageVersion Version_7_0 = new RazorLanguageVersion(7, 0);
 
-    public static readonly RazorLanguageVersion Latest = Version_7_0;
+    public static readonly RazorLanguageVersion Version_8_0 = new RazorLanguageVersion(8, 0);
```

Comment:
Do we know whether the default is `latest` or `7`? If the latter, where do we need to update the default?

Reply:
The default is `latest` as far as I can see in Razor (judging by C# references to `RazorLanguageVersion` and MSBuild property `RazorLangVerision`). We might need to update SDK, though? https://github.com/dotnet/sdk/blob/main/src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

Reply:
Yeah, looks like it: https://github.com/dotnet/sdk/blob/main/src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets#L46-L91. Let's file a bug to track that work.

Reply:
https://github.com/dotnet/sdk/issues/31058